### ISSUE TITLE
[UI suggestions] window Toolbar suggestions 

### ISSUE THREAD

**Issue by CatsArmy** [Kesomannen/gale#118]
unsure of how that icon was implemented but if its just like any other text with icons in it should be fine to just replace it if you also think this tweak makes sense
it would definitely need to have a tooltip on hover for the people who try and find it and probably wont hurt to have the same options in the settings menu or at least a button for opening that context menu 

i am not familiar with the codebase so if it would require a big refactor to add the feature its understandable if you wont or don't want to.

other changes could be that right clicking the settings icon would open up the File context menu instead of the button in the toolbar.

you could also replace the export from the toolbar with the export a modpack by just a bit changes:
just like you do with the settings of the multi launch in direct mode you could gray out the options for the export modpack and just use a Enum selector for the mode of exporting instead of a tab for just the profile code/file

the import window toolbar could be replaced with right clicking the profile icon for the managing profile button

and on and on i don't want to try and suggest my vison of what i think could improve the UI but in other words you could just replace the windows toolbar with the sidebar and settings menu + replace the home button with the gale icon as it makes more sense as showing the change log only would make it seem like that is the release notes and help/feedback home

also you could just expand the profile selector a bit more horizontally and and a edit icon to edit the name then 99% of all toolbar options would be redundant 

also these are just suggestions of what i think would improve the UI like most of my suggestions(GitHub issues) which are at the end of the day just what i think its understandable if you disagree with something/everything i suggest about the UI/UX of the app

**Comment by Kesomannen** [comment]
This is too disorganized and hard to follow, could you open another issue with a more clear suggestion?

**Comment by CatsArmy** [comment]
Yeah sure I'll do that later though gotta agree with ya that is a bit hard to read
